rpcs3/Crypto/unpkg.cpp

```diff
@@ -67,7 +67,7 @@ bool package_reader::read_header()
 	}
 
 	pkg_log.notice("Path: '%s'", m_path);
-	pkg_log.notice("Header: pkg_magic = 0x%x = \"%s\"", +m_header.pkg_magic, std::string(reinterpret_cast<const char*>(&m_header.pkg_magic), 4));
+	pkg_log.notice("Header: pkg_magic = 0x%x = \"%s\"", +m_header.pkg_magic, std::string_view(reinterpret_cast<const char*>(&m_header.pkg_magic + 1), 3)); // Skip 0x7F
```

Comment:
Why?
We log all this nonsense for a reason

Reply:
0x7f is an invalid character and it gets marked in my text editor. It's logged as hex anyway so I don't see the problem.